src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -1669,41 +1739,9 @@ public static partial class Kernel32
         ///         <see cref="GetLastError" />.
         ///     </para>
         /// </returns>
-        [DllImport(nameof(Kernel32), SetLastError = true, CharSet = CharSet.Unicode)]
+        [DllImport(nameof(Kernel32), CharSet = CharSet.Auto, SetLastError = true)]
```

Comment:
Why change the charset ? We try to not use Auto anywhere as it's equivalent to Unicode and just less clear.